app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_cp = command.split()
+    if len(command_cp) != 3 or command_cp[0] != "cp":
+        return
+
+    cp, source, destination = command_cp
+
+    if source == destination:
+        return
```

Comment:
try not to use blank return here, make a backward condition

Reply:
Please fix as per previous comments - remove unnecessary prints and returns and nest logic under if condition